sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/transaction/WalletTransactionEncoder.kt

```diff
@@ -121,7 +123,8 @@ internal class WalletTransactionEncoder(
         usk: UnifiedSpendingKey,
         amount: Zatoshi,
         toAddress: String,
-        memo: ByteArray? = byteArrayOf()
+        memo: ByteArray? = byteArrayOf(),
+        useZip317Fees: Boolean = false
```

Comment:
This is the stale comment, but the reasoning about why I like to avoid default arguments is still valid and applies to my other comment about default arguments.

Reply:
I agree with you in general. However, in all of those stale and now-removed changes, I was following the existing style (as I do everywhere in the Kotlin codebase, because I am not a Kotlin developer and am mostly fumbling in the dark here). If the memo argument did not have a default, I would not have added a default for the ZIP 317 fees flag, so that indicates the memo argument's default should definitely be removed from these internals.

Reply:
Yes this is a place where the existing style should get revised.  We're slowly working towards that! 😄